### DISCUSSION TITLE
臉書計劃在歐盟招聘1萬名員工 以推動元宇宙開發

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#99] [General]
最近爆發的元宇宙話題想必大家都有耳聞，臉書17日宣布未來5年要在歐盟招募1萬名員工發展元宇宙，可見祖克柏即將在未來實現他一直說的：「The Metaverse is the next Internet」
其實相關的話題如NFT、前陣子跟Apple槓上的Epic開發的要塞英雄，都是元宇宙的一部分，可預視未來世界將會是虛實交錯，充滿AR、VR。
除了這篇報導，有興趣的同學可以去圖書館看這期的商周，封面就是一個大大的元宇宙！
[臉書計劃在歐盟招聘1萬名員工 以推動元宇宙開發](https://news.cnyes.com/news/id/4747515)

**Comment by Jie2222** [comment]
Nvidia執行長黃仁勳預測，元宇宙的經濟規模，終將大過實體世界

**Comment by enoch-lightning** [comment]
之前的電影—一級玩家，應該就是元宇宙的概念。

**Reply by vvvioletta** [reply]
沒錯！
